test/OpenTelemetry.Exporter.Geneva.Tests/UnixUserEventsDataTransportTests.cs

```diff
@@ -0,0 +1,334 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if NET6_0_OR_GREATER
+
+#nullable enable
+
+using System.Diagnostics;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using Microsoft.LinuxTracepoints.Provider;
+using OpenTelemetry.Tests;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OpenTelemetry.Exporter.Geneva.Tests;
+
+[Trait("CategoryName", "Geneva:user_events:metrics")]
+public class UnixUserEventsDataTransportTests
+{
+    /*
+     * Instructions for running these tests:
```

Comment:
Thanks for writing this steps! 
My own linux kernel with user-events is broken, so not testing it myself, but will comeback to this as soon as kernel is updated in wsl!